src/modules/ShortcutGuide/ShortcutGuide.Ui/ShortcutGuideXAML/MainWindow.xaml

```diff
@@ -0,0 +1,100 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<winuiex:WindowEx
```

Comment:
Why use Winuiex??

Reply:
Makes window manipulation much easier.

Reply:
Anything specific? that is not in winui/wasdk?

Reply:
https://github.com/microsoft/PowerToys/pull/40834/files#diff-44a2780fd190a9e27dde92e7abac21e802e61ad7f921a67055e2981c05664635R56-R59

For example these lines are not as easely doable like with WinuiEx